components/datetime/src/options/length.rs

```diff
@@ -98,6 +99,27 @@ impl Default for Bag {
     }
 }
 
+impl Bag {
+    /// Constructs a Bag with all fields set to None
+    ///
+    /// Note that the [`Default`] implementation returns long date and time options
```

Comment:
Thought: This is kind-of odd. We should try hard to keep `Default::default()` consistent with `::new()` or `::empty()`.

Reply:
Yeah I agree. I didn't want to change things in this PR but perhaps we shoudl file a followup?